src/test/java/AlertsTest.java

```diff
@@ -0,0 +1,78 @@
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static constants.LocatorsConstants.*;
+import static constants.MainConstants.LINK_SELENIUM_ALERT;
+import static org.testng.Assert.assertEquals;
+
+public class AlertsTest {
+    private WebDriver driver;
+
+    @BeforeClass
+    void setup () {
+        ChromeOptions handlingSSL = new ChromeOptions();
+        handlingSSL.setAcceptInsecureCerts(true);
+        driver = new ChromeDriver(handlingSSL);
+        driver.get(LINK_SELENIUM_ALERT);
+    }
+
+    @Test
+    void testConfirmBoxAccept() {
+        WebElement confirmBox = driver.findElement(LOCATOR_CONFIRM_BOX);
+        confirmBox.click();
+
+        Alert alert = driver.switchTo().alert();
+        String alertText = alert.getText();
+        System.out.println("Alert text: " + alertText);
+
+        alert.accept();
+
+        WebElement confirmMessage = driver.findElement(LOCATOR_CONFIRM_BOX_MESSAGE);
+
+        assertEquals(confirmMessage.getText(), "You pressed OK!", "The confirm message must be \"You pressed OK!\"");
```

Comment:
move expected result to constant on class level

Reply:
for assertEquals assertion message "Confirmation message mismatch" is pretty enough. testng will log expected and actual result in case of fail